examples/metalk8s-solution-example/get_resource.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Not sure this is the best place to put this script as it's more related to CI than examples.
Should be put under `eve/`